www/js/overlay.js

```diff
@@ -245,27 +304,18 @@ const changeGurbanifontSizeButton = resizeButtonFactory(
 );
 const changeOpacityButton = resizeButtonFactory(increaseOpacity, decreaseOpacity);
 
-const themeSelector = document.querySelector('.theme-selector');
+textControls.append(controlsFactory([gurbaniColor, changeGurbanifontSizeButton], 'Gurbani'));
+textControls.append(separatorY());
+textControls.append(controlsFactory([textColor, changefontSizeButton], 'Text'));
+textControls.append(separatorY());
+textControls.append(controlsFactory(backgroundColor, 'BG'));
+textControls.append(separatorY());
+textControls.append(controlsFactory(changeBarSizeButton, 'Size'));
+textControls.append(controlsFactory(changeOpacityButton, 'Opacity'));
```

Comment:
should opacity not affect logo?

Reply:
![image](https://user-images.githubusercontent.com/2982784/69353078-cc144500-0c32-11ea-82ae-f19b4351688e.png)